students/kahyee/Session11/generator.py

```diff
@@ -0,0 +1,58 @@
+
+
+def intsum():
+    """
+    generator function that increments by the next integer starting at 0
+    """
+    i = 0
+    sum_value = 0
+    while True:
+        yield sum_value
+        i += 1
+        sum_value += i
+        
+def doubler():
+    """
+    generator function that becomes twice the value of the previous iteration
+    """
+    dbl_value = 1
+    while True:
+        yield dbl_value
+        dbl_value *= 2
+        
+def fib():
+    """
+    generator function of the fibonacci sequence starting with 1
+    """
+    prev_value = 0
+    cur_value = 1
+    intermediary = 0
+    
+    while True:
+        yield cur_value
+        intermediary = prev_value
+        prev_value = cur_value
+        cur_value += intermediary
+
+
+def isPrime(n):
```

Comment:
I like how you brought this out and made its own function. Would you be willing to show this to the class?